libprio_wrap.c

```diff
@@ -3196,11 +3196,19 @@ PyObject* PrioTotalShare_write_wrapper(const_PrioTotalShare p) {
 SECStatus PrioPacketVerify1_read_wrapper(PrioPacketVerify1 p, const unsigned char *data, unsigned int len, const_PrioConfig cfg) {
     SECStatus rv = SECFailure;
     msgpack_unpacker upk;
-    bool result = msgpack_unpacker_init(&upk, len+1);
+    // Initialize the unpacker with a reasonably sized initial buffer. The
+    // unpacker will eat into the initially reserved space for counting and may
+    // need to be reallocated.
+    bool result = msgpack_unpacker_init(&upk, 128);
```

Comment:
I guess this file is autogenerated based on the `.i`?

Reply:
Yes, I've found it useful to include the diff of the generated file.